app/include/user_modules.h

```diff
@@ -59,6 +60,7 @@
 #define LUA_USE_MODULES_SPI
 //#define LUA_USE_MODULES_STRUCT
 //#define LUA_USE_MODULES_SWITEC
+//#define LUA_USE_MODULES_TCA9535
```

Comment:
So sorry :) I'm new  for github and open source developing. In a short time I will learn rules ;)